stable/elasticsearch-exporter/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Elasticsearch stats exporter for Prometheus
 name: elasticsearch-exporter
-version: 3.2.0
+version: 3.2.1
```

since its a new feature, should probably increase the minor instead:

```suggestion
version: 3.3.0
```

---

Sure.